_layouts/collection.html

```diff
@@ -123,47 +124,70 @@ <h2 class="text-uppercase text-white font-family-condensed-extra font-size-h2 fl
-          aspect-ratio=0.5625 alt="{{ item.title }}"></crds-image>
+          <crds-image class="full-width"
+            src="{% if item.podcast.image.url %}{{ item.podcast.image.url  | imgix: site.imgix }}{% else %}{{ item.image.url | imgix: site.imgix }}{% endif %}?{{ site.imgix_params.placeholder_sixteen_nine }}"
+            aspect-ratio=0.5625 alt="{{ item.title }}"></crds-image>
         </div>
         <div class="col-sm-6">
-          <p class="text-uppercase text-orange font-size-smaller font-family-base-black flush-bottom">{{ page.title }}</p>
+          <p class="text-uppercase text-orange font-size-smaller font-family-base-black flush-bottom">{{ page.title }}
+          </p>
           <h2 class="text-uppercase text-white font-family-condensed-extra font-size-h2 flush-top">{{ item.title }}</h2>
           <p class="text-white font-size-smaller">{{ item.description }}</p>
         </div>
-       </a>
-      </div>
+      </a>
+    </div>
     {% endfor %}
     {% endif %}
   </div>
 </section>
 
 {% if page.trailer_video %}
-  <div aria-labelledby="Trailer modal" class="modal fade" id="trailerModal" role="dialog" tabindex="-1">
-    <div class="modal-dialog" role="document">
-      <div class="modal-content">
-        <div class="modal-header hard-bottom"><button aria-label="Close" class="close pull-right" data-dismiss="modal"
-            type="button"><span aria-hidden="true">&times;</span></button>
-        </div>
-        <div class="modal-body text-left push-ends hard-ends">
-            {% assign video_id = page.trailer_video | youtube_id %}
-            {% include _youtube-embed.html %}
-        </div>
-        <div class="modal-footer">{{ page.title }} Trailer</div>
+<div aria-labelledby="Trailer modal" class="modal fade" id="trailerModal" role="dialog" tabindex="-1">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header hard-bottom"><button aria-label="Close" class="close pull-right" data-dismiss="modal"
+          type="button"><span aria-hidden="true">&times;</span></button>
       </div>
+      <div class="modal-body text-left push-ends hard-ends">
+        {% assign video_id = page.trailer_video | youtube_id %}
+        {% include _youtube-embed.html %}
+      </div>
+      <div class="modal-footer">{{ page.title }} Trailer</div>
     </div>
   </div>
+</div>
 
-  <script type="text/javascript">
-    $('#trailerModal').on('hidden.bs.modal', function () {
-      $("#trailerModal iframe").attr("src", $("#trailerModal iframe").attr("src"));
+<script type="text/javascript">
+  $('#trailerModal').on('hidden.bs.modal', function () {
+    $("#trailerModal iframe").attr("src", $("#trailerModal iframe").attr("src"));
   });
-  </script>
+</script>
 {% endif %}
+
+<style>
+  .sign-up-button {
+    border: 1px solid black;
+    color: black;
+    padding: 10px 20px;
+    text-decoration: none;
+    display: inline-block;
```

Comment:
My only suggestion is to set this to `display: block` and some right-margin to center the button on mobile:
<img width="1305" alt="Screen Shot 2023-10-19 at 1 03 32 PM" src="https://github.com/crdschurch/crds-net/assets/58494322/6d86fec1-5b58-415f-ae6e-2f2ddd161d46">